app/src/main/java/com/example/android/navigation/GameWonFragment.kt

```diff
@@ -33,8 +34,13 @@ class GameWonFragment : Fragment() {
         val binding: FragmentGameWonBinding = DataBindingUtil.inflate(
                 inflater, R.layout.fragment_game_won, container, false)
         binding.nextMatchButton.setOnClickListener() {
-            view?.findNavController()?.navigate(R.id.action_gameWonFragment_to_gameFragment)
+            view?.findNavController()?.navigate(GameWonFragmentDirections.actionGameWonFragmentToGameFragment())
         }
+        displayArgs()
         return binding.root
     }
+    private fun displayArgs(){
```

Comment:
What should I do in order to see this toast?

Reply:
I know it's a bit weird but you should answer correctly to all the questions. You can see this toast on GameWonFragment.

Reply:
But each answer which I tap on will tell me that it's wrong. 
I never passed over the first question.